src/main/java/io/leonis/torch/Torch.java

```diff
@@ -0,0 +1,41 @@
+package io.leonis.torch;
+
+import com.googlecode.lanterna.TextColor;
+import com.googlecode.lanterna.gui2.*;
+import com.googlecode.lanterna.screen.*;
+import com.googlecode.lanterna.terminal.*;
+import java.nio.charset.Charset;
+import java.util.function.*;
+
+import io.leonis.torch.component.ZosmaBackground;
+
+/**
+ * The Class Torch.
+ *
+ * @author Jeroen de Jong
+ */
+public final class Torch {
+  /**
+   * Constructs a new window manager.
+   */
+  public Torch(final Function<WindowBasedTextGUI, WindowBasedTextGUI> GUI) {
+    try {
+      final Terminal terminal = new DefaultTerminalFactory(System.out, System.in, Charset.forName("UTF8")).createTerminal();
+      Screen screen = new VirtualScreen(new TerminalScreen(terminal));
+      screen.startScreen();
```

Comment:
@ThomasHakkers You researched click behavior in the terminal. what should be done to add it ?